src/components/Board/Board.component.js

```diff
@@ -104,7 +107,71 @@ export class Board extends Component {
 
     this.state = {
       openTitleDialog: false,
-      titleDialogValue: props.board && props.board.name ? props.board.name : ''
+      titleDialogValue: props.board && props.board.name ? props.board.name : '',
+      runLiveHelp: false,
+      runLiveUnlock: false,
+      hideCloseButton: true,
```

Comment:
I think this property can be deleted? I don't think we use it..

Reply:
Yes, of course, it was forgotten. Deleted!